lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:markdown] || {})
+        end
```

Comment:
I feel handling default options in different layers (`SummaryProcessor` and here) is complex. IMO removing this from `Filters::Greenmat` and handling default contexts in `Processor` and `SummaryProcessor` is simpler. (Maybe [this](https://github.com/increments/qiita-markdown/pull/30/files#r64180373) should also be done in that case)

What do you think?